src/azure-cli/azure/cli/command_modules/feedback/custom.py

```diff
@@ -370,6 +370,13 @@ def _build_issue_info_tup(command_log_file=None):
     format_dict["cli_version"] = _get_az_version_summary()
     format_dict["python_info"] = "Python {}".format(platform.python_version())
     platform_info = "{} (Cloud Shell)".format(platform.platform()) if in_cloud_console() else platform.platform()
+
+    try:
+        import distro
+        platform_info = '{}, {}'.format(platform_info, distro.name(pretty=True))
```

Comment:
`pretty=True` will make the name include version and codename (except for Ubuntu).

https://github.com/python-distro/distro/blob/c613bdfe2ae4f8b823e38376f537b1d07bac6bd3/distro.py#L264-L265

```py
def name(pretty: bool = False) -> str:
    """
    Return the name of the current OS distribution, as a human-readable
    string.
    If *pretty* is false, the name is returned without version or codename.
    (e.g. "CentOS Linux")
    If *pretty* is true, the version and codename are appended.
    (e.g. "CentOS Linux 7.1.1503 (Core)")
```